rm-config/defaults/keymap.toml

```diff
@@ -0,0 +1,46 @@
+[general]
+keybindings = [
+  { on = "?", action = "ShowHelp" },
+  { on = "F1", action = "ShowHelp" },
+
+  { on = "q", action = "Quit" },
+  { on = "Esc", action = "SoftQuit" },
+  { on = "Enter", action = "Confirm" },
+  { on = " ", action = "Select" },
+  { on = "Tab", action = "SwitchFocus" },
+  { on = "/", action = "Search" },
+  
+  { on = "1", action = "SwitchToTorrents" },
+  { on = "2", action = "SwitchToSearch" },
+
+  { on = "Home", action = "GoToBeginning" },
+  { on = "End", action = "GoToEnd" },
+  { on = "PageUp", action = "ScrollPageUp" },
+  { on = "PageDown", action = "ScrollPageDown" },
```

Comment:
can we use a format similar to https://github.com/d99kris/nchat#confignchatkeyconf ? i feel like it's a little more intuitive, but this format is also okay if it's too inconvenient to switch

Reply:
All available actions are already in the default config, so all you would ever want to do is just to change the key or add a modifier.